docs/source/markdown/podman-build.1.md

```diff
@@ -188,6 +188,10 @@ The [username[:password]] to use to authenticate with the registry if required.
 If one or both values are not supplied, a command line prompt will appear and the
 value can be entered.  The password is entered without echo.
 
+**--decryption-key** *key[:passphrase]*
+
+The [key[:passphrase]] to be used for decryption of images. Key can point to keys and/or certificates. Decryption will be tried with all keys. If the key is protected by a passphrase, it is required to be passed in the argument and omitted otherwise.
```

Comment:
For sake of consistency I recommend to keep the line break at 80 chars.

Reply:
We don't have --decryption-key until the next vendor of buildah